Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 var dep: [Package.Dependency] = [
     .package(
         url: "https://github.com/apple/swift-system",
-        from: "1.5.0"
+        exact: "1.5.0"
```

Comment:
I'm guessing you're pinning to 1.5.0 to avoid the regression introduced on Focal? Suggest adding a comment to that effect if so; "exact" is generally discouraged